src/main/java/io/gravitee/policy/ipfiltering/IPFilteringPolicy.java

```diff
@@ -170,8 +173,10 @@ private void processFilteredLists(final List<String> filteredList, final List<St
     private void fail(PolicyChain policyChain, String remoteAddress) {
         policyChain.failWith(
             PolicyResult.failure(
+                IPFILTERING_FORBIDDEN,
                 HttpStatusCode.FORBIDDEN_403,
-                "Your IP (" + remoteAddress + ") or some proxies whereby your request pass through are not allowed to reach this resource."
+                "Your IP (" + remoteAddress + ") or some proxies whereby your request pass through are not allowed to reach this resource.",
+                Maps.<String, Object>builder().put("address", remoteAddress).build()
```

Comment:
If we need to use remote address in response template, it's already available using `{#request.remoteAddress}`.
So maybe there is no need to add extra parameter here.
Could we remove it ?